htdocs/core/class/extrafields.class.php

```diff
@@ -883,7 +883,7 @@ public function fetch_name_optionals_label($elementtype, $forceload = false)
 					}
 
 					$this->attributes[$tab->elementtype]['type'][$tab->name] = $tab->type;
-					$this->attributes[$tab->elementtype]['label'][$tab->name] = $tab->label;
+					$this->attributes[$tab->elementtype]['label'][$tab->name] = $langs->transnoentities($tab->label);
```

Comment:
This denaturate the content on array  $this->attributes that should contains the database content.
Can't you make the translation later when we forge the error message ?